src/main/java/at/hannibal2/skyhanni/features/garden/composter/ComposterOverlay.kt

```diff
@@ -433,6 +438,48 @@ class ComposterOverlay {
         return first ?: error("First is empty!")
     }
 
+    private fun getMaterials(internalName: String, itemName: String, itemsNeeded: Double) {
+        val sackData = fetchSackItem(internalName.asInternalName())
+        val amountInSacks = sackData.amount
+        val sacksLoaded = sackData.outdatedStatus
```

Comment:
You can use this oneliner here: `val (amountInSacks, _, sacksLoaded) = ...`

Reply:
But these val's are being set to different values?